README.md

```diff
@@ -17,9 +17,10 @@ The extension type can be :
 
 - [Node.js](http://nodejs.org/) with [NPM](https://www.npmjs.com/), Minimum version 6.9.5 LTS (https://nodejs.org/)
 - [Yeoman](http://yeoman.io/) : npm install -g yo
-- [Grunt](https://www.npmjs.com/package/grunt) : npm install -g grunt
 - [Tfx-cli](https://www.npmjs.com/package/tfx-cli) : npm install -g tfx-cli
 - [typescript (necessary for new hub extension)](https://www.npmjs.com/package/typescript) : npm install -g typescript
+- [tslint](https://www.npmjs.com/package/tslint)  : npm install -g tslint
```

Comment:
We need to reduce the number of pre-reqs the user must install. Only Node, Yeoman, and this generator package should be required. The rest can be listed as dev dependencies. NPM scripts can find the executables for dependencies, so "TFX" and "tsc" don't even need to be globally installed to be used.